File: abc/abc184/a.rs
// https://atcoder.jp/contests/abc184/submissions/30510426

use proconio::input;

fn main() {
    input! {
        a: i32,
        b: i32,
        c: i32,
        d: i32
    }
    println!("{}", a * d - b * c);
}

File: abc/abc184/a.cs
// https://atcoder.jp/contests/abc184/submissions/46503078

using System;

public class A
{
  static void Main()
  {
    string[] t = Console.ReadLine().Split(" ");
    int a = int.Parse(t[0]);
    int b = int.Parse(t[1]);
    t = Console.ReadLine().Split(" ");
    int c = int.Parse(t[0]);
    int d = int.Parse(t[1]);

    Console.WriteLine(a * d - b * c);
  }
}

